OrbitQt/main.cpp

```diff
@@ -26,13 +26,17 @@ int main(int argc, char* argv[]) {
   const std::string dump_path = Path::GetDumpPath();
 #ifdef _WIN32
   const char* handler_name = "crashpad_handler.exe";
+  const std::string crash_server_url =
+      "https://clients2.google.com/cr/staging_report";
 #else
   const char* handler_name = "crashpad_handler";
+  const std::string crash_server_url =
+      "http://clients2.google.com/cr/staging_report";
```

Comment:
Is there a general limitation with crashpad that the linux client does not support https?

Reply:
This check occurs in `https://github.com/chromium/crashpad/blob/master/util/net/http_transport_socket.cc` which included only for linux, fuchsia and android and only if `!(crashpad_is_in_fuchsia || (crashpad_is_linux && crashpad_is_in_chromium))`

Reply:
Since we use crashpad/20191009 this restriction applies to all linux clients. If we update crashpad version to newer one (after 14.10.19) then it will apply only for linux that are not in chromium.

Reply:
Default value which looks into crashpad_dependencies could be overwritten by passing the `crashpad_use_boringssl_for_http_transport_socket=true`.